web/themes/custom/itkdev/itkdev_project_theme/templates/components/webformSubmissionReportHeader.html.twig

```diff
@@ -0,0 +1,34 @@
+{% if blockers|length > 0 %}
+  <div class="blockers flex alert-danger p-3 rounded">
+    <div class="m-5 self-center text-xl">
+      <i class="fa-solid fa-triangle-exclamation"></i>
+    </div>
+    <div>
+      {% for blocker in blockers %}
+        <div>
+          {{ blocker['#text_stop']|raw }}
+        </div>
+      {% endfor %}
+    </div>
+  </div>
+{% endif %}
+
+{# Specific to law track webform submissions #}
+{% if tool.getToolData['webform_submission:' ~ tool.id.value].webform|keys|first is same as ('maa_vi') %}
```

Comment:
Consider adding a `isLawTrack`, say, function to the tool helper to make this easier to read and maintain, i.e. something like

```
{% if tool.isLawTrack() %}
```

Reply:
Are you absolutely sure what the first webform key is? Can we pass an instance of `WebformSubmission` to the template to make these kinds of checks easier to make?

Reply:
It's not really a check on track type but rather a check on a very specific webform id.

Added comment to explain why the assumption is fair.